### ISSUE TITLE
bug: hide menu at runtime, nav to the last item and nav down once more, will move the cursor to the top

### ISSUE THREAD

**Issue by luwenjin** [davetcc/tcMenu#285]
version: 3.0.0

How to reproduce: 
- open simpleU8g2 project
- add these two lines to the end of `setup()`
```
menuStartToasting.setVisible(false);
menuMgr.notifyStructureChanged();
```
- run the new program
- nav down to the bottom, hit "down" again, the cursor will be on the first item

**Comment by davetcc** [comment]
https://github.com/davetcc/tcMenuLib/issues/193

**Comment by davetcc** [comment]
Library issue fixed for 3.2
